Info param doc (#42)
agama-project/agama-project.github.io: docs/user/boot_options/index.md

  agama.config_url=usb:///agama.yaml
  ```

- `agama.info`
  Points agama to info file that contains additional agama settings. It is useful when more options

>   are provided to agama and typing during boot is unreliable.

Reviewer: ```suggestion
  more options for Agama and typing them during boot is not possible.
```

  ```text
  agama.info=ftp://myserver/devel.info
  ```

  :::warning

>   Available from Agama12.

Reviewer: I would omit this, because we are not putting this information for other options.

Author: yeap, but on other hand as first beta for SLES is out, we should somehow start indicating when it is available.

  :::

  :::warning

>   Supports only parameters specified on this section. Kernel parameters are ignored.

Reviewer: I do not think it is a warning. You can make it part of the description of the options or, if you want to emphasize this, let's use a simple `note`.
